test/testinput/bufrtest_bufrdescription.yaml

```diff
@@ -0,0 +1,74 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+
+bufr:
+  obsdatain: "./Data/gdas.t12z.1bmhs.tm00.bufr_d"
+
+  mnemonicSets:
+    - mnemonics: [SAID, FOVN, YEAR, MNTH, DAYS, HOUR, MINU, SECO, CLAT, CLON, CLATH, CLONH, HOLS]
+    - mnemonics: [SAZA, SOZA, BEARAZ, SOLAZI]
+    - mnemonics: [TMBR]
+      channels : 1-15
```

Comment:
The BUFR  file defined in "obsdatain" is mhs.  So, the channels here should be 1-5

Reply:
I believe you are correct :) For the purpose of this example it doesn't matter too much. The NCEPLib-BUFR library returns 0's for the unused channels. I'd been switching the input files between amsua and mhs, which is why its a weird value.